shapely/svgtree.py

```diff
@@ -0,0 +1,55 @@
+"""svgtree.py: ElementTree based geometry SVG"""
+
+import itertools
+from xml.etree.ElementTree import Element, ElementTree, SubElement
+
+
+class SVGTree(ElementTree):
+    @classmethod
+    def path(cls, coords):
+        return "M {} L {}".format(
+            f"{coords[0][0]},{coords[0][1]}",
+            " L ".join(f"{xy[0]},{xy[1]}" for xy in coords[1:]),
+        )
+
+    @classmethod
+    def fromgeom(cls, geom):
+        """Returns Element"""
+        if geom.geom_type == "Point":
+            return cls.frompoint(geom)
+        elif geom.geom_type == "LineString":
+            return cls.fromlinestring(geom)
+        elif geom.geom_type == "LinearRing":
+            return cls.fromlinearring(geom)
+        elif geom.geom_type == "Polygon":
+            return cls.frompolygon(geom)
+        else:
+            raise NotImplementedError
+
+    @classmethod
+    def frompoint(cls, geom):
```

Comment:
Seems like `path` and the individual geometry functions should all be private in order to keep the public API smaller.